src/styles/repair.css

```diff
@@ -0,0 +1,13 @@
+.submit {
```

Comment:
Since we're using tailwind, I think we should convert whatever uses this to tailwind. We can make it reusable by wrapping it in a component (like a reusable button)

Reply:
No longer used. Can remove

Reply:
removed